src/main/java/mk/p/singleton/v1/MySingleton.java

```diff
@@ -0,0 +1,13 @@
+package mk.p.singleton.v1;
+
+public class MySingleton {
+    private MySingleton() {
+        System.out.println("Singleton created");
+    }
+
+    public static final MySingleton INSTANCE = new MySingleton();
```

Comment:
Why this field is defined between methods?

Reply:
Coz it is a new style.

Reply:
Review java code convention doc